### ISSUE TITLE
Current master fails "make update"

### ISSUE THREAD

**Issue by mouse07410** [openssl/openssl#11714]
MacOS 10.15.4, Xcode-11.4.1, perl-5.28.2, OpenSSL - current master.

Fails on `make depend`:
```
$ ./config --prefix=/Users/ur20980/openssl-3 --debug --openssldir=/Users/ur20980/openssl-3/etc --with-rand-seed=rdcpu,os enable-aria enable-ec_nistp_64_gcc_128 enable-md2 enable-rc5 enable-weak-ssl-ciphers zlib-dynamic enable-ssl-trace enable-trace enable-buildtest-c++ threads
Operating system: x86_64-apple-darwinDarwin Kernel Version 19.4.0: Wed Mar 4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64
WARNING! If you wish to build 32-bit libraries, then you have to
         invoke 'KERNEL_BITS=32 ./config  '--prefix=/Users/ur20980/openssl-3' '--debug' '--openssldir=/Users/ur20980/openssl-3/etc' '--with-rand-seed=rdcpu,os' 'enable-aria' 'enable-ec_nistp_64_gcc_128' 'enable-md2' 'enable-rc5' 'enable-weak-ssl-ciphers' 'zlib-dynamic' 'enable-ssl-trace' 'enable-trace' 'enable-buildtest-c++' 'threads''.
Configuring OpenSSL version 3.0.0-alpha2-dev for target darwin64-x86_64-cc
Creating configdata.pm
Running configdata.pm
Creating Makefile

**********************************************************************
***                                                                ***
***   OpenSSL has been successfully configured                     ***
***                                                                ***
***   If you encounter a problem while building, please open an    ***
***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
***   and include the output from the following command:           ***
***                                                                ***
***       perl configdata.pm --dump                                ***
***                                                                ***
***   (If you are new to OpenSSL, you might want to consult the    ***
***   'Troubleshooting' section in the INSTALL.md file first)      ***
***                                                                ***
**********************************************************************
+ make update
( cd .; /opt/local/bin/perl5.28 VMS/VMSify-conf.pl \
				< apps/openssl.cnf > apps/openssl-vms.cnf )
( cd .; /opt/local/bin/perl5.28 crypto/bn/bn_prime.pl > crypto/bn/bn_prime.h )
( cd .; /opt/local/bin/perl5.28 crypto/objects/objects.pl -n \
				crypto/objects/objects.txt \
				crypto/objects/obj_mac.num \
				> crypto/objects/obj_mac.new && \
	    mv crypto/objects/obj_mac.new crypto/objects/obj_mac.num )
( cd .; /opt/local/bin/perl5.28 crypto/objects/objects.pl \
				crypto/objects/objects.txt \
				crypto/objects/obj_mac.num \
				> include/openssl/obj_mac.h )
( cd .; /opt/local/bin/perl5.28 crypto/objects/obj_dat.pl \
				include/openssl/obj_mac.h \
				> crypto/objects/obj_dat.h )
( cd .; /opt/local/bin/perl5.28 crypto/objects/objxref.pl \
				crypto/objects/obj_mac.num \
				crypto/objects/obj_xref.txt \
				> crypto/objects/obj_xref.h )
( cd .;	cat crypto/objects/obj_compat.h >> include/openssl/obj_mac.h )
( cd .; /opt/local/bin/perl5.28 crypto/conf/keysets.pl \
			        > crypto/conf/conf_def.h )
( cd .; /opt/local/bin/perl5.28 crypto/asn1/charmap.pl \
			        > crypto/asn1/charmap.h )
( cd .; /opt/local/bin/perl5.28 fuzz/mkfuzzoids.pl \
				crypto/objects/obj_dat.h \
				> fuzz/oids.txt )
( b=`pwd`; set -e; cd .; \
          /opt/local/bin/perl5.28 util/ck_errf.pl -strict -internal; \
          /opt/local/bin/perl5.28 -I$b util/mkerr.pl  -internal )
EVP reason code 208 collision at EVP_R_SET_DEFAULT_PROPERTY_FAILURE
make: *** [errors] Error 25
```

Output of `perl configdata.pm --dump`:
```
$ perl configdata.pm --dump

Command line (with current working directory = .):

    /opt/local/bin/perl5.28 ./Configure darwin64-x86_64-cc --prefix=/Users/ur20980/openssl-3 --debug --openssldir=/Users/ur20980/openssl-3/etc --with-rand-seed=rdcpu,os enable-aria enable-ec_nistp_64_gcc_128 enable-md2 enable-rc5 enable-weak-ssl-ciphers zlib-dynamic enable-ssl-trace enable-trace enable-buildtest-c++ threads

Perl information:

    /opt/local/bin/perl5.28
    5.28.2 for darwin-thread-multi-2level

Enabled features:

    aria
    asm
    async
    autoalginit
    autoerrinit
    autoload-config
    bf
    blake2
    buildtest-c++
    camellia
    capieng
    cast
    chacha
    cmac
    cmp
    cms
    comp
    ct
    deprecated
    des
    dgram
    dh
    dsa
    dso
    dtls
    dynamic-engine
    ec
    ec2m
    ecdh
    ecdsa
    ec_nistp_64_gcc_128
    engine
    err
    filenames
    fips
    gost
    idea
    legacy
    makedepend
    md2
    md4
    mdc2
    module
    multiblock
    nextprotoneg
    pinshared
    ocb
    ocsp
    padlockeng
    pic
    poly1305
    posix-io
    psk
    rc2
    rc4
    rc5
    rdrand
    rfc3779
    rmd160
    scrypt
    secure-memory
    seed
    shared
    siphash
    siv
    sm2
    sm3
    sm4
    sock
    srp
    srtp
    sse2
    ssl
    ssl-trace
    static-engine
    stdio
    tests
    threads
    tls
    trace
    ts
    ui-console
    whirlpool
    weak-ssl-ciphers
    zlib
    zlib-dynamic
    tls1
    tls1-method
    tls1_1
    tls1_1-method
    tls1_2
    tls1_2-method
    tls1_3
    dtls1
    dtls1-method
    dtls1_2
    dtls1_2-method

Disabled features:

    afalgeng            [not-linux]      OPENSSL_NO_AFALGENG
    asan                [default]        OPENSSL_NO_ASAN
    crypto-mdebug       [default]        OPENSSL_NO_CRYPTO_MDEBUG
    devcryptoeng        [default]        OPENSSL_NO_DEVCRYPTOENG
    egd                 [default]        OPENSSL_NO_EGD
    external-tests      [default]        OPENSSL_NO_EXTERNAL_TESTS
    fuzz-libfuzzer      [default]        OPENSSL_NO_FUZZ_LIBFUZZER
    fuzz-afl            [default]        OPENSSL_NO_FUZZ_AFL
    ktls                [default]        OPENSSL_NO_KTLS
    msan                [default]        OPENSSL_NO_MSAN
    sctp                [default]        OPENSSL_NO_SCTP
    ubsan               [default]        OPENSSL_NO_UBSAN
    unit-test           [default]        OPENSSL_NO_UNIT_TEST
    uplink              [no uplink_arch] OPENSSL_NO_UPLINK
    ssl3                [default]        OPENSSL_NO_SSL3
    ssl3-method         [default]        OPENSSL_NO_SSL3_METHOD

Config target attributes:

    AR => "ar",
    ARFLAGS => "r",
    CC => "cc",
    CFLAGS => "-g -O0 -Wall",
    HASHBANGPERL => "/usr/bin/env perl",
    RANLIB => "ranlib -c",
    RC => "windres",
    asm_arch => "x86_64",
    bn_ops => "SIXTY_FOUR_BIT_LONG",
    build_file => "Makefile",
    build_scheme => [ "unified", "unix" ],
    cflags => "-arch x86_64",
    cppflags => "-D_REENTRANT",
    defines => [ "OPENSSL_BUILDING_OPENSSL", "ZLIB", "ZLIB_SHARED" ],
    disable => [  ],
    dso_scheme => "dlfcn",
    enable => [  ],
    includes => [  ],
    lflags => "-Wl,-search_paths_first",
    lib_cflags => "",
    lib_cppflags => "-DL_ENDIAN",
    lib_defines => [  ],
    module_cflags => "-fPIC",
    module_cxxflags => undef,
    module_ldflags => "-bundle",
    perl_platform => "Unix",
    perlasm_scheme => "macosx",
    shared_cflag => "-fPIC",
    shared_defines => [  ],
    shared_extension => ".\$(SHLIB_VERSION_NUMBER).dylib",
    shared_ldflag => "-dynamiclib -current_version \$(SHLIB_VERSION_NUMBER) -compatibility_version \$(SHLIB_VERSION_NUMBER)",
    shared_rcflag => "",
    shared_sonameflag => "-install_name \$(INSTALLTOP)/\$(LIBDIR)/",
    shared_target => "darwin-shared",
    sys_id => "MACOSX",
    thread_defines => [  ],
    thread_scheme => "pthreads",
    unistd => "<unistd.h>",

Recorded environment:

    AR = 
    ARFLAGS = 
    AS = 
    ASFLAGS = 
    BUILDFILE = 
    CC = clang
    CFLAGS = -Os -Ofast -std=gnu17 -march=native -g
    CPP = 
    CPPDEFINES = 
    CPPFLAGS = 
    CPPINCLUDES = 
    CROSS_COMPILE = 
    CXX = clang++
    CXXFLAGS = -std=gnu++17 -Os -Ofast -march=native -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
    HASHBANGPERL = 
    LD = 
    LDFLAGS = 
    LDLIBS = 
    MT = 
    MTFLAGS = 
    OPENSSL_LOCAL_CONFIG_DIR = 
    PERL = 
    RANLIB = 
    RC = 
    RCFLAGS = 
    RM = 
    WINDRES = 
    __CNF_CFLAGS = 
    __CNF_CPPDEFINES = 
    __CNF_CPPFLAGS = 
    __CNF_CPPINCLUDES = 
    __CNF_CXXFLAGS = 
    __CNF_LDFLAGS = 
    __CNF_LDLIBS = 

Makevars:

    AR              = ar
    ARFLAGS         = r
    CC              = clang
    CFLAGS          = -Os -Ofast -std=gnu17 -march=native -g
    CPPDEFINES      = 
    CPPFLAGS        = 
    CPPINCLUDES     = 
    CXX             = clang++
    CXXFLAGS        = -std=gnu++17 -Os -Ofast -march=native -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
    HASHBANGPERL    = /usr/bin/env perl
    LDFLAGS         = 
    LDLIBS          = 
    PERL            = /opt/local/bin/perl5.28
    RANLIB          = ranlib -c
    RC              = windres
    RCFLAGS         = 

NOTE: These variables only represent the configuration view.  The build file
template may have processed these variables further, please have a look at the
build file for more exact data:
    Makefile

build file:

    Makefile

build file templates:

    Configurations/common0.tmpl
    Configurations/unix-Makefile.tmpl
    Configurations/common.tmpl
```

**Comment by levitte** [comment]
Fix exists in #11708.

**Comment by slontis** [comment]
This should be addressed now so closing this.
